source/GameData.cpp

```diff
@@ -1355,16 +1370,18 @@ void GameData::PrintShipTable()
 
 
 
-void GameData::PrintWeaponTable()
+void GameData::PrintGunTable()
 {
 	cout << "name" << '\t' << "cost" << '\t' << "space" << '\t' << "range" << '\t'
-		<< "energy/s" << '\t' << "heat/s" << '\t' << "recoil/s" << '\t'
-		<< "shield/s" << '\t' << "hull/s" << '\t' << "push/s" << '\t'
-		<< "homing" << '\t' << "strength" <<'\n';
+		 << "reload/s" << '\t' << "energy/s" << '\t' << "heat/s" << '\t'
+         << "recoil/s" << '\t' << "shield/s" << '\t' << "hull/s" << '\t'
+         << "push/s" << '\t' << "heat dmg/s" << '\t' << "slow/s" << '\t'
+         << "disruption/s" << '\t' << "ion/s" << '\t' << "homing" << '\t'
+         << "strength" << '\t' << "velocity" << '\n';
 	for(auto &it : outfits)
 	{
-		// Skip non-weapons and submunitions.
-		if(!it.second.IsWeapon() || it.second.Category().empty())
+		// only iterate over guns
```

Comment:
Comments are expected to be formatted sentences, and are to explain the intent of the code (vs. restating what the code lines say they do).